lib/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -1154,13 +1154,8 @@ protected function prepare_links( $post ) {
 					continue;
 				}
 
-				if ( 'post_tag' === $tax ) {
```

Comment:
I realize that. I just see more special casing to accommodate the `rest_base` mistake seeping into the code and it makes me cringe. This code doesn't even use `rest_base` from a constant or some other variable -- it just replicates it. If I'm a jerk for pointing this out, then I apologize.